### ISSUE TITLE
Pressing enter after a query sometimes casues stack smashing which triggers a sigabrt and data loss.

### ISSUE THREAD

**Issue by sandersantema** [monkoose/fzf-hoogle.vim#13]
Once in a while I experience a stack smashing error after pressing with a search query, unfortunately the error message is quite bare and I haven't been able to reliable trigger the error. My neovim simply immediately quits and the shell reports a stack smashing error and neovim being terminated by a sigabrt. Would you have any clue how I'd be able to gather more information to debug this error? I saw a commit for fixing an error related to empty searches, this is not what causes this error given that I've experienced the stack smashing error with searches which obviously do yield results such as searching for `sort`.

Luckily I haven't had any big problems but I think this is quite a severe bug given that it causes data loss. This is because when neovim is forcefully killed by the sigabrt nothing is saved nor is it recoverable by undofile.

Some version info which might be related:
`hoogle 5.0.18`
`NVIM v0.5.0-dev+933-g4620ad604`
`Tmux 3.1c-2.1`
`fish 3.1.2-1744-ga4e2a3c04`

I suspect this might have something to do with `hoogle` given that a stack smashing error is related to `gcc` and given that your plugin runs as vimscript I wouldn't expect it to trigger a stack smashing error, haskell which compiles to c would obviously be a much more likely culprit. Although I do suspect the way `hoogle` is triggered would cause this error.

**Comment by sandersantema** [comment]
Here is a screenshot of the bug which I experienced again just a few minutes ago:
![image](https://user-images.githubusercontent.com/21057514/102266192-65865700-3f18-11eb-913d-88ef73fa4103.png)

**Comment by sandersantema** [comment]
Another one:
![image](https://user-images.githubusercontent.com/21057514/102266638-04ab4e80-3f19-11eb-87cb-e515e86a66b7.png)

**Comment by monkoose** [comment]
I believe it is something with fish shell, can you try to reproduce it in any other shell on your system?

I never had this issue on my side (i use zsh), not sure how to fix it either because `enter` just call another fzf internal `run` function.
You don't have this problem with any other fzf plugins? Maybe fzf or fish issue trackers are better places to ask about this problem.

**Comment by sandersantema** [comment]
I don't have this problem with any other plugin no. I'll try running from another shell for a while, that should at least narrow it down to a problem with specific to the shell.

**Comment by sandersantema** [comment]
Unfortunately that didn't take too long :( It doesn't seem to be specific problem with fish, I triggered the stack smashing running bash `5.0.18` too:
![image](https://user-images.githubusercontent.com/21057514/102353931-17ba2f00-3faa-11eb-80f6-d6fbfbcb7793.png)

I did discover that it only happens when I'm running inside of tmux. When in tmux I can reliably trigger stack smashing by just performing a lot of bogus search queries in quick succession i.e. pressing `asdf` then `enter` a lot but this doesn't happen outside of tmux.
